app/src/main/java/org/mozilla/focus/tips/TipManager.kt

```diff
@@ -55,11 +57,27 @@ class Tip(
             val id = R.string.tip_fresh_look
             val name = context.getString(R.string.app_name)
 
+            val sumoTopic = if (AppConstants.isKlarBuild) {
+                SupportUtils.SumoTopic.WHATS_NEW_KLAR
+            } else {
+                SupportUtils.SumoTopic.WHATS_NEW_FOCUS
+            }
+
+            val whatsNewUrl = getSumoURLForTopic(context, sumoTopic)
+            val deepLink = {
+                context.components.tabsUseCases.addTab(
+                    url = whatsNewUrl,
+                    source = SessionState.Source.Internal.Menu,
```

Comment:
The source is not the menu, but `Internal.HomeScreen`. It's wrong for the other tips too, so Ieave it up to you if you want to fix them all here or in a separate PR. 🙂